frontend/app/utils/locale-utils.server.ts

```diff
@@ -1,8 +1,25 @@
+import { createCookie } from '@remix-run/node';
+
+import { parse, serialize } from 'cookie';
 import { type Namespace, createInstance } from 'i18next';
 import I18NexFsBackend from 'i18next-fs-backend';
 import { resolve } from 'node:path';
 
-import { getLocale } from '~/utils/locale-utils';
+export function createLangCookie() {
+  const cookieName = '_gc_lang';
+
+  const langCookie = createCookie(cookieName, {
```

Comment:
We shouldn't use `createCookie` because overriding `parse` and `serialize` doesn't work as you thought. `parseOptions` and `serializeOptions` are always underfined based on my tests. We should use `parse` and `serialize` func from `cookie` package for that specific cookie because it needs to work differently then Remix's cookies. The value of the cookie should be base64 encoded in our case.

Reply:
Acknowledged; will fix.

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/pull/17/files#diff-73af98351d32bbf3e05728a23721407c32230588067302df2571f014a28bc9d9R21-R30)

No longer making a call to Remix's `createCookie(..)` function. Instead, I am directly returning a `Cookie` type.